argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="enable-support-for-namespace-scoped-argo-rollouts-installation"]
+= Enable support for namespace-scoped Argo Rollouts installation
+:context: enable-support-for-namespace-scoped-argo-rollouts-installation
+
+toc::[]
+
+By default, the {gitops-title} installation enables support for the cluster-scoped Argo Rollouts installation. In this mode of installation, the Argo Rollouts custom resources (CRs) defined in any namespace are reconciled by that Argo Rollouts instance. This allows you to use Argo Rollouts CR across any namespace on the cluster.
+
+The namespace-scoped Argo Rollouts installation is an alternative to the cluster-scoped Argo Rollouts installation. This mode of installation does not require cluster-wide permissions by using `cluster roles` or `cluster role binding`. This means that only users with permissions that are limited to a specific namespace can use such resources.
+
+A namespaced-scoped instance provides security benefits by limiting the cluster scope of a single Argo Rollouts instance. With this mode of installation, Argo Rollouts only handles Argo Rollouts CR within the same namespace.
+
+For example, if Argo Rollouts is installed in the 'argo-rollouts-ns' namespace, the Argo Rollouts installation only handles Argo Rollouts CR in the 'argo-rollouts-ns' namespace. The Argo Rollouts CR installed in another namespace, for example, the 'my-application' namespace, are not used.
+
+To configure namespace-scoped or cluster-scoped Argo Rollouts installation, you must use the `NAMESPACE_SCOPED_ARGO_ROLLOUTS` environment variable, which is set under the `.spec.config.env` property of the {gitops-title} Subscription CR. When this environment variable is set to `true`, Argo Rollouts provides support for namespace-scoped Argo Rollouts installation and when it is set to `false`, Argo Rollouts supports cluster-scoped  installation.
```

Comment:
I would add this final sentence to this paragraph. It's important to be specific about the default:

If the value is not set, it defaults to `false` (cluster-scoped).

Reply:
This comment has been fixed as per the reviewer's comment. Fixed.